src/test/groovy/org/prebid/server/functional/AuctionSpec.groovy

```diff
@@ -1,13 +1,23 @@
 package org.prebid.server.functional
 
+import org.prebid.server.functional.model.db.StoredRequest
 import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.PrebidStoredRequest
+import org.prebid.server.functional.service.PrebidServerService
+import org.prebid.server.functional.util.PBSUtils
+import spock.lang.Shared
 import spock.lang.Unroll
 
 import static org.prebid.server.functional.util.SystemProperties.PBS_VERSION
 
 class AuctionSpec extends BaseSpec {
 
+    private static final int MAX_TIMEOUT = 5000
+    private static final int DEFAULT_TIMEOUT = PBSUtils.getRandomNumber(1000, MAX_TIMEOUT)
```

Comment:
There is a constant for min timeout in base spec. Also there is a method in the amp spec to get random timeout. Move that method to the base class and use that.